app/assets/stylesheets/sdg/goals/show.scss

```diff
@@ -63,6 +63,21 @@
         }
       }
     }
+
+    .goal-description-content {
+
+      .goal-description {
+
+        &.short {
```

Comment:
SCSS Lint is giving us a warning: Nesting should be no greater than 4, but was 5.

One option here:

```scss
    .goal-description-content {

      .read-more {
        text-align: center;
      }
    }

    .goal-description {

      &.short {
        height: rem-calc(150);
        overflow: hidden;
      }
    }
```

Another option:

```scss
    .goal-description {

      &.short {
        height: rem-calc(150);
        overflow: hidden;
      }

      + .read-more {
        text-align: center;
      }
    }
```